.gitignore

```diff
@@ -1 +1,2 @@
-vulcanized.html
+bower_components
+.DS_Store
```

Comment:
Hmm, ignoring `bower_components` is project specific so it should stay in `.gitignore`, but it's probably better to ignore `.DS_Store` in the user's global ignore. Will fix.